srcpkgs/tpm2-tss/template

```diff
@@ -0,0 +1,35 @@
+# Template file for 'tpm2-tss'
+pkgname=tpm2-tss
+version=2.3.1
+revision=1
+build_style=gnu-configure
+configure_args="--disable-weakcrypto --with-crypto=gcrypt"
+hostmakedepends="autoconf-archive automake libtool pkg-config
+ doxygen libltdl-devel"
+makedepends="libltdl-devel libgcrypt-devel"
+short_desc="OSS implementation of the TCG TPM2 Software Stack"
+maintainer="Nathan Owens <ndowens04@gmail.com>"
+license="BSD-2-Clause"
+homepage="https://github.com/tpm2-software/tpm2-tss"
+distfiles="https://github.com/tpm2-software/tpm2-tss/archive/${version}.tar.gz"
+checksum=da818b8cb82630f561ed088ef3204b243a8f1de680623a09d775deb6014c4c99
+
+pre_configure() {
+	./bootstrap
+}
+
+post_install() {
+	vlicense LICENSE
+}
+
+tpm2-tss-devel_package() {
+	depends="${sourcepkg}>=${version}_${revision}"
```

Comment:
I guess this is incomplete. anyway why do you use libgcrypt?

Reply:
It wants to use OpenSSL and leave it to try and use LibreSSL it gives compilation issues, like RAND_OpenSSL undefined references.

Incomplete as in?

Reply:
Does it need `depends+="$makedepends"`?

Reply:
Unsure on the depends+=$makedepends Every subpackage I seen used depends the way I have it.

Reply:
Checked its header, it doesn't need `makedepends`